.github/workflows/cron-checks.yml

```diff
@@ -213,7 +223,9 @@ jobs:
 
   automated-code-review:
     name: Automated Code Review
-    runs-on: macos-13
+    runs-on: macos-12
```

Comment:
Shouldn't this be macOS-14?

Reply:
I thought it might make sense to run a code review on older Xcode so it can catch more issues. Plus not overload macos-14 runners queue (if there is one).